app/services/zendesk_drop_off_service.rb

```diff
@@ -37,15 +38,26 @@ def create_ticket_and_attach_file
           EitcZendeskInstance::SIGNATURE_METHOD => @drop_off.signature_method
       }
     )
-    attach_file_and_save_ticket(ticket)
+    success = ticket.save
+    raise ZendeskServiceHelper::ZendeskAPIError.new("Error creating drop off ticket: #{ticket.errors}") unless success
```

Comment:
We can just use `save!` to let the Zendesk Ruby API raise a good specific error, and then we wouldn't have to check for success.

Reply:
https://github.com/zendesk/zendesk_api_client_rb/wiki/Errors#resources